### ISSUE TITLE
[Bug]: eslint does not work

### ISSUE THREAD

**Issue by vlkv** [aspect-build/rules_lint#656]
### What happened?

I am desperately trying to make eslint work via aspect's rules_lint. I had 3 attempts:
- in monorepo at my work
- in [my fork of basel_starters/js](https://github.com/vlkv/bazel_starters_js/tree/jslib_jsapp_to_try_eslint) (their repo has everything configured but there is no 'hello world/cowsay' app there, so in my fork I have added it)
- finally I have lost faith in my bazel skills and decided to run some examples from the rules_lint/example (should have do it in the first place). Created an [MRE in my fork](https://github.com/aspect-build/rules_lint/pull/655) because I had to add some minor flags+toolkits+bazelisk stuff there.

In all these three usecases eslint does not detect problems in my js/ts code, such as unused variable.

### Version

Development (host) and target OS/architectures:

linux amd64

Output of `bazel --version`:
```
./bazelisk --version
bazel 7.5.0
```

Version of the Aspect rules, or other relevant rules from your
`WORKSPACE` or `MODULE.bazel` file:
```
bazel_dep(name = "aspect_rules_lint", version = "0.0.0")
bazel_dep(name = "aspect_rules_js", version = "2.0.0")
bazel_dep(name = "aspect_rules_swc", version = "2.0.0")
bazel_dep(name = "aspect_rules_ts", version = "3.0.0")
```

Language(s) and/or frameworks involved:

JavaScript/TypeScript, Eslint

### How to reproduce

Use [this fork of rules_lint](https://github.com/aspect-build/rules_lint/pull/655) which shows the MRE.

```shell
cd ./example
./bazelisk build //src:ts
```

### Any other information?

Actual result NO ERRORS from eslint.
Expected result: there SHOULD BE SOME ERRORS from eslint! Because they present in the [example/src/file.ts](https://github.com/aspect-build/rules_lint/blob/main/example/src/file.ts)

**Comment by alexeagle** [comment]
Are you differentiating between errors and warnings? The eslint exit code is what determines whether Bazel will fail. Since you're running a `bazel build` nothing is going to fail for something like Unused variable for example, I'd expect is typically a warning.
